pkgs/ffigen/doc/errors.md

```diff
@@ -0,0 +1,23 @@
+# Errors in ffigen
+
+This file documents various errors and their potential fixes related to ffigen.
+
+## Errors in source header files
+
+Under the hood ffigen uses libclang to parse header files. Any compiler warnings/errors should be logged (with SEVERE level), these should be resolved as it can potentially generate *incomplete* or *invalid* bindings that might cause silent errors at runtime.
```

Comment:
FFIgen uses libclang to parse header files. Any compiler warnings/errors should be logged (with SEVERE level).
Compiler errors and warnings should be resolved as they can potentially generate *invalid* bindings that might cause silent errors and crashes at runtime.